src/frontend/src/components/SnackBar/SnackBar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Snackbar from '@material-ui/core/Snackbar';
+import IconButton from '@material-ui/core/IconButton';
+import CloseIcon from '@material-ui/icons/Close';
+
+export default function SimpleSnackbar() {
+  const [open, setOpen] = React.useState(true);
```

Comment:
Most of our code does `userState(true)` vs. `React.useState(true)`.  We should clean this up in another PR to make them all the same.  This is fine for now.